src/debugger/files.md

```diff
@@ -6,4 +6,4 @@ This is a useful feature, which is not found in other debuggers, the functionali
 
 So, at any time in the debugging session you can replace the stdio file descriptors to use network sockets created by rizin, or replace a network socket connection to hijack it.
 
-This functionality is also available in r2frida by using the dd command prefixed with a backslash. In rizin you may want to see the output of dd? for proper details.
+This functionality is also available in `r2frida` by using the dd command prefixed with a backslash. In rizin you may want to see the output of `dd?` for proper details.
```

Comment:
Remove this line since we don't have `r2frida`, nor our own Frida plugin just yet.